mmv1/templates/terraform/pre_create/alloydb_instance.go.erb

```diff
@@ -1,3 +1,18 @@
+// Temporarily remove the enablePublicIp field if it is set to true since the
+// API prohibits creating instances with public IP enabled.
```

Comment:
Currently because of security. We plan on updating the API in the upcoming weeks to enforce some security requirements for creating the resource with `enabledPublicIP = true`. I was suggested to add this in for now to avoid undesirable behavior by Stephen